### PR TITLE
New members page 

### PR DESCRIPTION
**Pull request by manohar9999**
For issue https://github.com/cncf/sig-security/issues/226

### CODE REVIEW

**Comment by manohar9999** [comment]
@ultrasaurus please review

File: NEWMEMBERS.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## New Members
+
+New members are advised to:
+
+* Join the [CNCF Slack team](https://slack.cncf.io/), particulary [#sig-security](https://cloud-native.slack.com/messages/CDJ7MLT8S) channel and introduce yourself.
+* Initially go through the following documents in the repository:
+	* [README.md](https://github.com/cncf/sig-security/blob/master/README.md) 
+	* [CODE-OF-CONDUCT.md](https://github.com/cncf/sig-security/blob/master/CODE-OF-CONDUCT.md)
+	* [usecases.md](https://github.com/cncf/sig-security/blob/master/usecases.md)
+* Regularly join the [Zoom meeting](https://github.com/cncf/sig-security/blob/master/README.md#meeting-time) atleast for the first couple of months to get yourself upto speed.
```
Comment (ultrasaurus) [inline NEWMEMBERS.md]: typos?
 atleast => at least
 upto => up to

File: NEWMEMBERS.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	* [usecases.md](https://github.com/cncf/sig-security/blob/master/usecases.md)
+* Regularly join the [Zoom meeting](https://github.com/cncf/sig-security/blob/master/README.md#meeting-time) atleast for the first couple of months to get yourself upto speed. 
+* Here are multiple ways to get involved:
+	* Join the meeting as advised above and express your area of interests or if you want to work on any specific issue.
+	* Express your thoughts or ask questions on an issue you find interesting.
+	* Choose an issue where [help is needed](https://github.com/cncf/sig-security/labels/help%20wanted) and comment on it expressing interest. 
+
+Peer support:
+
+* Every new member will be assigned a buddy preferably in the same time zone.
```
Comment (ultrasaurus) [inline NEWMEMBERS.md]: I think we need to have more structure around the buddy system... can you split into a separate PR, so we can make sure we actually have a group of folks willing to be buddies before we promise this?

File: NEWMEMBERS.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+* The buddy will help in identifying the new member's areas of interest thus assigning them to the right peer group. 
+
+Purpose of this plan:
+
+* Ensure you become familiar with the team, know how you will contribute and what you will learn and deliver.
```
Comment (ultrasaurus) [inline NEWMEMBERS.md]: Can you move this to the top?  I think it would be nice if it were folded into an introductory sentence or two, and link to the mission/charter so people know what we're all working toward!

File: NEWMEMBERS.md (hunk 2 above)
Comment (manohar9999) [reply NEWMEMBERS.md]: I removed it from the doc, will create a separate PR.

File: NEWMEMBERS.md (hunk 3 above)
Comment (manohar9999) [reply NEWMEMBERS.md]: Added this as well.
